honeybee_radiance/cli/glare.py

```diff
@@ -0,0 +1,104 @@
+"""honeybee radiance dcglare command."""
+import click
+import sys
+import logging
+import os
+
+from honeybee_radiance.config import folders
+from honeybee_radiance_command.dcglare import Dcglare, DcglareOptions
+
+
+_logger = logging.getLogger(__name__)
+
+
+@click.group(help='Commands to run dcglare in Radiance.')
+def dcglare():
+    pass
+
+
+@dcglare.command('two-phase')
+@click.argument(
+    'dc-direct', type=click.Path(exists=True, dir_okay=False, resolve_path=True))
+@click.argument(
+    'dc-total', type=click.Path(exists=True, dir_okay=False, resolve_path=True))
+@click.argument(
+    'sky-mtx', type=click.Path(exists=True, dir_okay=False, resolve_path=True))
+@click.argument(
+    'view-rays', type=click.Path(exists=True, dir_okay=False, resolve_path=True)
+)
+@click.option(
+    '--glare-limit', '-l', type=float,
+    help='When this option is provided, the program calculates glare autonomy, where '
+    'any DGP value at or above the limit indicates the presence of glare. If the option '
+    'is not provided, the program calculates DGP under each sky condition in the sky '
+    'matrix instead.'
+)
+@click.option(
+    '--threshold-factor', '-b', type=float,
+    help='If larger than 100, it is used as constant threshold in cd/m2. If less than '
+    'or equal to 100, this factor multiplied by the average luminance in each view will '
+    'be used as threshold for detecting the glare sources (not recommended). The '
+    'default value is 2000 (fixed threshold method).'
+)
+@click.option(
+    '--occupancy-schedule', '-sf', type=click.Path(exists=True, dir_okay=False, 
+    resolve_path=True), 
```

Comment:
In order to use this in a recipe and keep it optional, you have to relax the `exists=True` input so you can deal with it inside the command itself.